music_this_week_app/backend/eventFinder.py

```diff
@@ -147,4 +147,4 @@ def __init__(self, event_dict):
         self.date = event_dict['start_time']  # "2016-07-19 19:30:00"
         self.date = datetime.strptime(self.date, "%Y-%m-%d  %H:%M:%S")
     def __repr__(self):
-        return "Title: %r \nVenue: %r" % (self.title, self.venue_name)
\ No newline at end of file
+        return "Title: %r \nVenue: %r" % (self.title, self.venue_name)
```

Comment:
Please include an empty line at the end of the file
https://www.reddit.com/r/Python/comments/1zjugg/question_why_does_pep8_recommend_leaving_a_blank/